raft/raft_test.go

```diff
@@ -616,22 +616,20 @@ func testLeaderElectionOverwriteNewerLogs(t *testing.T, preVote bool) {
 	}
 	// This network represents the results of the following sequence of
 	// events:
-	// - Node 1 won the election in term 1.
-	// - Node 1 replicated a log entry to node 2 but died before sending
-	//   it to other nodes.
-	// - Node 3 won the second election in term 2.
-	// - Node 3 wrote an entry to its logs but died without sending it
-	//   to any other nodes.
+	// - Node 1 starts the first election at term 1 and fails.
```

Comment:
The old description was correct: it's telling how we could get to the configuration created by newNetworkWithConfig, not what is about to happen in this test.

Reply:
@ping40 with comment from @bdarnell above I guess we should close this PR? Thanks!